ffmpeg-plugin/mcm_video_tx.c

```diff
@@ -29,85 +28,102 @@ typedef struct McmVideoMuxerContext {
     enum AVPixelFormat pixel_format;
     AVRational frame_rate;
 
-    mcm_conn_context *tx_handle;
-} McmVideoMuxerContext;
+    MeshClient *mc;
+    MeshConnection *conn;
+
+ } McmVideoMuxerContext;
```

Comment:
should teher be whitespace before "}" ?

Reply:
Having a blank line here is not prohibited by the C standard. Sometimes I add blank lines to make the structure definition look better in the editor. So, this is to make the code easier to read.